src/ducks/resource.js

```diff
@@ -33,75 +33,58 @@ const resourceReducer = (state = initialState, action) => {
 };
 
 // Action Creators
-const fetchResource = (id, type) => {
+function fetchResource(id, type) {
   type = type || 'projects';
-  return (dispatch) => {
+  return (dispatch, getState) => {
+    const project = getState().project.data;
```

Comment:
In the container: https://github.com/zooniverse/pandora/blob/master/src/containers/ProjectDashboardContainer.jsx#L42 and then uses it in the redux `connect` here https://github.com/zooniverse/pandora/blob/master/src/containers/ProjectDashboardContainer.jsx#L58